src/class_objectProcessor.py

```diff
@@ -749,28 +749,14 @@ def processmsg(self, data):
 
                 ackdata = genAckPayload(streamNumber, 0)
                 toAddress = '[Broadcast subscribers]'
```

Comment:
these two lines can be removed.

Reply:
we can not remove line 751 because it is used in below lines

Reply:
ok